snakes_cafe.py

```diff
@@ -276,6 +226,105 @@ def optional_menu():
 #             menu[unpacked[i][1]] = {unpacked[i][0]: [0, unpacked[i][1], unpacked[i][2]]}
 #     print(menu)
 
+
+class Order:
+    """
+    new class
+    """
+    def __init__(self):
+        self.default
+
+    def __str__(self):
+        return 'Welcome to Snakes Cafe'
+
+    def __len__(self):
+        counter = 0
+        for key, value in menu.items():
+            for k, v in value.items():
+                if v[0] != 0:
+                    counter += v[0]
+        return counter
+
+    def __repr__(self):
+        print('<Order #{} | Items: {} | Total: ${:.2f}>'.format(order_id, Order.__len__, subtotal * 1.101))
+
+    def add_item(order_line, order_quantity):
+        """
+        adds items to user order
+        """
+        global subtotal
+        for key, value in menu.items():
+            if order_line in value.keys():
```

Comment:
When iterating over a dictionary, you do not need to specify `.keys()` as the loop will automatically iterate over the keys.

Reply:
the .keys() part is in the "if" though, so it's not looping through the keys I thought ... but i will take it out and see if it works still